test/form/samples/optional-chaining/_config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	description: 'supports URL externals',
```

Comment:
It looks like this file does not fit the test

Reply:
Thanks, I've fixed this up.